### ISSUE TITLE
use name set in CCU for objekt name in iobroker

### ISSUE THREAD

**Issue by disaster123** [ioBroker/ioBroker.hm-rpc#1089]
**Is your feature request related to a problem? Please describe.**
I'm always frustated by not knowing which device it is - because i only see KEQ... in iobroker

**Describe the solution you'd like**
Simply use the name set in CCU as an object name in iobroker

**Describe alternatives you've considered**
none

**Additional context**
none

Thanks!

**Comment by foxriver76** [comment]
indeed, we were too, thus we developed the hm-rega Adapter. You should really try it out.

**Comment by foxriver76** [comment]
All Logic stuff and also names, variables, scripts etc are Not Provided over XML api unfortunately thus normally you Need Both Adapters.

**Comment by GermanBluefox** [comment]
As @foxriver76 wrote, the names are not available via the RPC interface. hm-rega reads them from the CCU and renames the objects of hm-rpc, so both adapters are needed.
